### DISCUSSION TITLE
Webinstaller can only flash without an inserted SD card

### DISCUSSION THREAD

**Discussion by uwen70** [jomjol/AI-on-the-edge-device#1974] [General]
The webinstaller's website should state, that an inserted SD Card must be removed before flashing. Otherwise the Camera Flash LED will flicker and the ESP32 module will not be found (Failed to initialize).

**Comment by jomjol** [comment]
Interesting - in my device this is not a problem. I can flash with inserted SD card

**Reply by uwen70** [reply]
If you can flash with inserted SD card depends on previously installed software. It probably depends on the addressing of the SD card. Hence the flickering of the LED on GPIO4. It is not clear to me why the web installer accesses the SD card at all.
